e2e/rook-test-framework/objects/environment_manifest.go

```diff
@@ -0,0 +1,23 @@
+package objects
+
+import (
+	"flag"
+)
+
+type EnvironmentManifest struct {
+	K8sVersion string
+	Platform   string
+	RookTag    string
+}
+
+func NewManifest() EnvironmentManifest {
+	e := EnvironmentManifest{}
+
+	flag.StringVar(&e.K8sVersion, "k8s_version", "v1.6", "Version of kubernetes to test rook in; v1.5 | v1.6 are the only version of kubernetes currently supported")
+	flag.StringVar(&e.Platform, "rook_platform", "Kubernetes", "Platform to install rook on; Kubernetes is the only platform currently supported")
+	flag.StringVar(&e.RookTag, "rook_tag", "quay.io/jbw976/rookd:v0.3.1-74-g0554a83-dirty", "Docker tag of the rook operator to install, must be in quay.io or local environment")
```

Comment:
also consider naming this `rook_version` instead of `rook_tag` because it actually means the entire fully qualified image repo, image name and image tag.